examples/extras/event_study.py

```diff
@@ -0,0 +1,32 @@
+from datetime import date
+import numpy as np
+import san
+
+import sys
+sys.path.append("../..")
```

Comment:
😨

Reply:
Open for suggestions how to write an example for something that is not released yet.

Reply:
It should be possible to install the module using `pip` from a local folder, so it should be possible to make the notebook work as if the module is published officially.